client/js/components/procedure/publicindex/DpSearchProcedureMap.vue

```diff
@@ -328,6 +328,26 @@ export default {
   },
 
   methods: {
+    fitToBounds () {
+      setTimeout(() => {
+        if (window.markersLayer.getLayers().length > 0) {
```

Comment:
When no procedures/markers are found at all this will poll forever, right?

Reply:
no. setTimeOut is only called once. https://developer.mozilla.org/en-US/docs/Web/API/setTimeout.
setIntervall is the one wich calls itself gain and again.

Reply:
indeed, mixed it up with setInterval